onnxruntime/python/onnxruntime_pybind_mlvalue.h

```diff
@@ -21,6 +21,9 @@ namespace python {
 
 namespace py = pybind11;
 
+static constexpr const char* PYTHON_ORTVALUE_OBJECT_NAME = "OrtValue";
+static constexpr const char* PYTHON_ORTVALUE_NATIVE_OBJECT_ATTR = "_ortvalue";
+
```

Comment:
Declaring statics in the header is not a good practice. Can you make it extern and declare it in the .cc file?

Reply:
Sure

Reply:
Hmm - I pushed a commit but it doesn't reflect here in the PR - it reflects in the branch when I look at its commits though